turbinia/workers/analysis/jenkins.py

```diff
@@ -163,7 +163,8 @@ def analyze_jenkins(version, credentials):
     credentials_registry = {hash: username for username, hash in credentials}
     # TODO: Add timeout parameter when dynamic configuration is ready.
     # Ref: https://github.com/google/turbinia/issues/244
-    weak_passwords = bruteforce_password_hashes(credentials_registry.keys())
+    weak_passwords = bruteforce_password_hashes(
```

Comment:
What does `-m 3200` do?  Could you add a short comment?

Reply:
Went an added in comments to all invocations

Reply:
Went and added in comments to all invocations